### DISCUSSION TITLE
Push Issue.

### DISCUSSION THREAD

**Discussion by ZoeQgOgO** [community/community#22968] [New to GitHub]
<p>I had a new repo on Github. After init, add and commit, I try to push. Here came the problem. No matter I push from my branch or master branch, it came out the same output.</p>
<p>Please help me with this!!</p>
<p>problem: </p>
<p>rror: RPC failed; curl 55 SSL_write() returned SYSCALL, errno = 10053</p>
<p>fatal: The remote end hung up unexpectedly</p>
<pre><code>Git LFS: (0 of 0 files, 2 skipped) 0 B / 0 B, 329.51 MB skipped Counting objects: 36971, done.
Delta compression using up to 2 threads.
Compressing objects: 100% (20720/20720), done.
efatal: The remote end hung up unexpectedlyiB | 5.37 MiB/s
rror: RPC failed; curl 55 SSL_write() returned SYSCALL, errno = 10053
Writing objects: 100% (36971/36971), 4.12 GiB | 10.28 MiB/s, done.
Total 36971 (delta 14718), reused 36966 (delta 14717)
fatal: The remote end hung up unexpectedly
Everything up-to-date
</code></pre>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/zoeqgogo">@zoeqgogo</a>!</p>
<p>I’m not sure what is causing this off the top of my head. I think we’re going to have to do some more in-depth troubleshooting. Could you <a href="https://github-debug.com/" rel="nofollow noopener">please follow these instructions on the GitHub Debug site</a>? In the ticket that you open in Step 3, please also include a link to this forum topic as that will help our technical troubleshooting team connect that new ticket with this issue.</p>
<p>Thanks!</p>

**Comment by ZoeQgOgO** [comment]
<p>Hi,</p>
<p>I solved the problem, just because of the size I posted. Everythings work well so far. Thanks for replying.</p>
<p>Best.</p>
